src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WebSocketTransportDuplexSessionChannel.cs

```diff
@@ -878,7 +873,7 @@ Message PrepareMessage(WebSocketReceiveResult result, byte[] buffer, int count)
                     Message message;
                     if (_useStreaming)
                     {
-                        using (var wrappedStream = new MaxMessageSizeStream(
+                        var wrappedStream = new MaxMessageSizeStream(
```

Comment:
Question only: does the removal of this 'using' indicate the MaxMessageSizeStream is being disposed somewhere else?